apps/sms/services/js/threads.js

```diff
@@ -164,5 +164,18 @@
     }
   };
 
+  Threads.Messages = {
+    get(id) {
+      id = +id;
+      var threadId = messageMap.get(id);
+
+      if (Threads.has(threadId)) {
+        return Threads.get(threadId).messages.get(id);
+      }
+
+      return null;
+    }
```

Comment:
mmm right, I think that in the caller I could use the threadId in addition to the messageId. But I thought that if the messageId is enough...

In the future DB-based service I think it should be possible to access a specific message directly.